README.md

```diff
@@ -185,9 +186,28 @@ If a published post on openliberty.io/blog contains an error or needs updating i
 
 ## Troubleshooting Asciidoc
 
-Certain characters (eg apostrophe ' ) in the main heading are displayed incorrectly. To fix, escape with a backslash (`\`).
+* Certain characters (eg apostrophe ' ) in the main heading are displayed incorrectly. To fix, escape with a backslash (`\`).
 eg `= Minimise turnaround times with Open Liberty\'s dev mode`
 
+* Syntax highlighting in code snippets isn't displaying correctly. To fix, make sure you've put your code snippet in `source` tags:
+
+   ```
+   [source,xml]
+   ----
+   your code snippet here
+   ----
+   ````
+   Where `xml` is the language used in your code snippet. See the [supported list of languages](https://docs.asciidoctor.org/asciidoctor/latest/syntax-highlighting/coderay/). If your languages isn't supported (eg a Dockerfile or shell script), remove the language attribute: `[source]`.
+
+* Screenshots look scruffy. To fix, place the custom border attribute on the line above the image tag for _all screenshots_:
+
+   ```
+   [.imageblock.img_border_light]
```

Comment:
I like that we are being more specific with the CSS selector `[.imageblock.img_border_light]` vs `[.img_border_light]`. 

Unfortunately, a lot of the existing blog posts use `[.imageblock.img_border_light]`. I suspect developers will look at existing `*.adoc` files instead of this README.md instruction to learn how to add a image border.

Reply:
Hmm, I think my templates are probably more of a culprit here... eg https://raw.githubusercontent.com/OpenLiberty/blogs/prod/templates/post-single-author.adoc I'll fix them if you prefer the more specific class @kinueng ?

Reply:
Thinking this more, I think I prefer `[.img_border_light]` so blog authors can focus more on the human read content rather than the code.  Authors may not be comfortable with CSS so minimizing the noise from code specific related things in the `adoc` file is a good thing.

Reply:
After further discussion, leaving as `[.img_border_light]` format.